src/main/kotlin/no/nav/syfo/oversikthendelse/retry/KOversikthendelseRetry.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.syfo.oversikthendelse.retry
+
+import java.time.LocalDateTime
+
+const val RETRY_OVERSIKTHENDELSE_LIMIT_HOURS = 24
+const val RETRY_OVERSIKTHENDELSE_INTERVAL_MINUTES = 15L
+const val RETRY_OVERSIKTHENDELSE_COUNT_LIMIT = (RETRY_OVERSIKTHENDELSE_LIMIT_HOURS / (RETRY_OVERSIKTHENDELSE_INTERVAL_MINUTES / 60.0)).toInt()
+
+data class KOversikthendelseRetry(
+    val created: LocalDateTime,
+    val retryTime: LocalDateTime,
+    val retriedCount: Int,
+    val fnr: String,
+    val oversikthendelseType: String,
+    val personOppgaveId: Int
+)
+
+fun KOversikthendelseRetry.hasExceededRetryLimit(): Boolean {
+    val retriedCount = this.retriedCount
+    return retriedCount >= RETRY_OVERSIKTHENDELSE_COUNT_LIMIT
+}
```

Comment:
Trenger man å assigne `retriedCount` først? Dette kan vel ev. gjøres om til en one-liner, hvis det ikke blir for utydelig hva som skjer?

Reply:
Endret til en linje nå.